### PR TITLE
Translate Ruby 2.5.0 release post (zh_cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

File: zh_cn/news/_posts/2017-12-25-ruby-2-5-0-released.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+lang: zh_cn
+---
+
+我们高兴地宣布 Ruby 2.5.0 已发布。
+
+Ruby 2.5.0 是 2.5 系列首个稳定版。此版本包括数个新特性和性能提升，例如：
+
+## 新特性
+
+- 允许在 do/end 中直接使用 rescue/else/ensure。[&#x5b;功能 #12906&#x5d;](https://bugs.ruby-lang.org/issues/12906)
```
Comment (stomar) [inline zh_cn/news/_posts/2017-12-25-ruby-2-5-0-released.md]: Escaping of square brackets is not necessary, see updated en version

**Comment by stomar** [comment]
Note: should be merged as single commit with current PR title as commit message.

**Comment by dsh0416** [comment]
brackets fixed

File: zh_cn/news/_posts/2017-12-25-ruby-2-5-0-released.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## 与 2.4 相比的重要变化
+
+- SecureRandom 倾向使用操作系统方法而非 OpenSSL。[[漏洞 #9569]]((https://bugs.ruby-lang.org/issues/9569))
+- 将 cmath, csv, date, dbm, etc, fcntl, fiddle, fileutils, gdbm, ipaddr, scanf, sdbm, stringio, strscan, webrick, zlib 从标准库中提升到默认 gem。
+- 升级 [Onigmo](https://github.com/k-takata/Onigmo/) 到 6.1.3。
+  - 添加 [缺位运算符](https://github.com/k-takata/Onigmo/issues/87)。
+  - 注意，Ruby 2.4.1 也包含这项改动。
+- 更新到 Psych 3.0.2。
+- 更新到 Rubygems 2.7.3。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2017-12-25-ruby-2-5-0-released.md]: `RubyGems`

File: zh_cn/news/_posts/2017-12-25-ruby-2-5-0-released.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- 更新到 RDoc 6.0.1。
+  - [词法分析程序由 IRB 更换为 Ripper](https://github.com/ruby/rdoc/pull/512)，极大提升了生成文档的速度。
+  - 修正过去这些年存在的缺陷。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2017-12-25-ruby-2-5-0-released.md]: Not sure about the accuracy, according to en doc. I'd go with "修正近十年来的大量缺陷"。

**Comment by dsh0416** [comment]
@hlcfan Problems mentioned have fixed.

**Comment by hlcfan** [comment]
Looking good, thanks!

@JuanitoFatas @AndorChen Do you want to merge and publish this?

**Pull request merged**